app/operations/handle_sms_blocklist_update.rb

```diff
@@ -0,0 +1,56 @@
+# Handle the list of blocked SMS numbers returned by the API, using pagination
+class HandleSmsBlocklistUpdate
+  include Dry::Monads[:result, :do, :try]
+  include EventSource::Command
+
+  def call(options = {})
```

Comment:
should we add validation for the options and ensure the :state and :event are present?

Reply:
We can - let me put in an update.